python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/antlr_asset_selection.py

```diff
@@ -0,0 +1,24 @@
+from antlr4 import CommonTokenStream, InputStream
+
+from dagster._annotations import experimental
+from dagster._core.definitions.antlr_asset_selection.generated.AssetSelectionLexer import (
+    AssetSelectionLexer,
+)
+from dagster._core.definitions.antlr_asset_selection.generated.AssetSelectionParser import (
+    AssetSelectionParser,
+)
+
+
+@experimental
+class AntlrAssetSelection:
```

Comment:
Might want to name this something that doesn't end with `AssetSelection`, as it sorta implies that this is a subclass of `AssetSelection`, which it is not.

Maybe `AntlrAssetSelectionProcessor` or something?

Also this doesn't need an `@experimental` tag because it's not exposed to the user